### PR TITLE
Update `jest-light-runner`

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| License                  | MIT

The only relevant changes for us are `--maxWorkers` and `--runInBand` support.

Ref https://github.com/babel/babel/pull/14486.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/14488"></a>

### CODE REVIEW

File: test/testSetupFile.js [hunk 1]
```diff
@@ -1 +0,0 @@
-jest.setTimeout(10000);
```
Comment (nicolo-ribaudo) [inline test/testSetupFile.js:1]: Older `jest-light-runner` versions did not support `setupFilesAfterEnv`, so this file was not loaded at all (and thus we don't really need it).

The latest `jest-light-runner` supports `setupFilesAfterEnv`, but it does not support `jest.setTimeout`.

**Comment by nicolo-ribaudo** [comment]
I'm merging this when it's green, since:
- it's just a devDep update
- already two people complained that without `--runInBand` support it's impossible to debug using the Node.js/Chrome debugger :grimacing:

**Review by jridgewell — APPROVED** [review]

**Review by existentialism — APPROVED** [review]

**Comment by nicolo-ribaudo** [comment]
The "E2E tests (breaking) / Test (babel)" failure might be related, I'll investigate tomorrow.

**Comment by nicolo-ribaudo** [comment]
Ok it was just a timeout, I restarted it and it passed.

**Pull request merged**